### PR TITLE
Added definitions for latlon-spherical.js

### PR DESCRIPTION
**Pull request by HandyG52**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/chrisveness/geodesy/blob/master/latlon-spherical.js>>
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by DenisCarriere — CHANGES_REQUESTED** [review]
Can't review these diffs, what's this PR changing?

**Comment by HandyG52** [comment]
Geodesy contains a latlon-spherical.js for which there were no types
defined.  I added them, and tests.

**Comment by DenisCarriere** [comment]
Check the changes, there's +340 additions and -268 deletions

**Comment by DenisCarriere** [comment]
@HandyG52 You should close this PR and redo it, in your Git commits you deleted `geodesy-test.ts` and then re-added, the git changes is really hard to follow.

👎 Close this PR
